### ISSUE TITLE
updates?

### ISSUE THREAD

**Issue by ghost** [prof7bit/TorChat#68]
Hey,

I noticed there's a lot of issues and pull requests and the project hast been updated for over a year is this still actively maintained?

**Comment by HostFat** [comment]
I suggest you to follow this other project (it isn't a Torchat client)
https://github.com/ricochet-im/ricochet

**Comment by ghost** [comment]
Thank you very much! installing right now on arch.

**Comment by GuyXY** [comment]
I've already seen Ricochet but it seems like NOBODY uses it...

**Comment by virtadpt** [comment]
Which obviously means that we need another secure and private encrypted messaging system.

**Comment by GuyXY** [comment]
That would be nice but I think it should be either compatible with the TorChat protocol and/or have new features like group chats, a server/client model etc.
That would have so much potential!

**Comment by ghost** [comment]
so why is this project no longer maintained?

**Comment by ikuraj** [comment]
+1 :+1: 
not sure why do people keep implementing their own stuff (and reinventing the wheel)--I think we need this project alive and kicking

**Comment by ghost** [comment]
@kaptoxic why don't we just make a new fork and ask people to submit there pull request or issues there. What do you think?

**Comment by ikuraj** [comment]
Yes, I agree--I think you can make a new fork that will be more active. At least, until the original author gets back to us--in the meantime, I will buy that man a :beer: :) :)

**Comment by ghost** [comment]
I'm so tempted to make an alternative to Torchat that would just use the same protocol, so it would be fully compatible with the original TC client.

**Comment by ikuraj** [comment]
Interesting: [tor messenger beta released](https://blog.torproject.org/blog/tor-messenger-beta-chat-over-tor-easily). Perhaps this might prove useful for fixing the code for pidgin plugin.

**Comment by timofonic** [comment]
Where's the new official fork? Why Tor Messenger got beta released and this code is dead?

Lots developers say LibPurple (@kaptoxic, pidgin is their GTK client using libpurple modular library) is full of horrible spaguetti code. I would like to know what others think about that.

https://github.com/TheTorProject/tor-messenger-build

What's this .patch messy HELL?
https://github.com/TheTorProject/tor-messenger-build/tree/master/projects/instantbird

Can anyone from @TheTorProject explain about this instead just providing shitty propaganda? It seems every NGO acts the same way, just another form of corporations :D
